README.md

```diff
@@ -18,21 +18,40 @@ To find out more about the Rabble Rouser project, check out our [documentation r
 
         git clone https://github.com/rabblerouser/core.git
 
-3. Start a Docker container to develop in (this also starts containers for dependent services):
+3. Install Docker and start its application.
```

Comment:
This is already described in Step 1 in the readme, including the links. We could have a step to verify that the docker daemon is running? 

3. Verify Docker is running

`$ docker info`

Reply:
Right, I did that step but then Docker wasn't running, and later instructions failed. By the time I realised the problem, I forgot that I'd installed Docker :-)

Updated this merge request to follow on from the existing “install Docker” instruction.